x-pack/plugins/endpoint/server/routes/resolver/children.ts

```diff
@@ -39,8 +39,9 @@ interface ChildrenPathParams {
 export const validateChildren = {
   params: schema.object({ id: schema.string() }),
   query: schema.object({
-    after: schema.maybe(schema.string()),
-    limit: schema.number({ defaultValue: 10, min: 1, max: 100 }),
+    children: schema.number({ defaultValue: 10, min: 10, max: 100 }),
```

Comment:
If the UI isn't specifying these params (children and generations) dynamically, we should consider hard coding it in the server. I don't see a reason why this should be dynamic.